server/lib/config/pbench-server-default.cfg

```diff
@@ -33,7 +33,7 @@ commit-id=unknown
 
 # WARNING - the pbench-server.cfg file should provide a definition of
 # pbench-top-dir, e.g.:
-#     pbench-top-dir = /srv/pbench
+#	pbench-top-dir = /srv/pbench
```

Comment:
Do we need this whitespace change?

Reply:
It was already there.

Reply:
But it is marked as a change, so *something* changed.